google-cloud-storage/src/main/java/com/google/cloud/storage/PostPolicyV4.java

```diff
@@ -121,7 +130,7 @@ public Builder setContentType(String contentType) {
         return this;
       }
 
-      public Builder Expires(String expires) {
+      public Builder setExpires(String expires) {
         fieldsMap.put("expires", expires);
```

Comment:
Looks like this was definitely missed. I'd recommend deprecating rather than renaming for both rename cases.